### ISSUE TITLE
Support multiple tilesets in the same map?

### ISSUE THREAD

**Issue by simicn** [RafaelBarbosatec/bonfire#50]
Bonfire is a great project, but it seems that bonfire doesn't  support multiple tilesets in the same map now.
I test with 0.6.15, and the map struct like this:

```
{ 
 "compressionlevel":-1,
 "height":20,
 "infinite":false,
 "layers":[
     ...
 ],
 "nextlayerid":3,
 "nextobjectid":1,
 "orientation":"orthogonal",
 "renderorder":"right-down",
 "tiledversion":"2020.06.25",
 "tileheight":16,
 "tilesets":[
        {
         "firstgid":1,
         "source":"world_set.json"
        }, 
        {
         "firstgid":2,
         "source":"outside_set.json"
 }],
 "tilewidth":16,
 "type":"map",
 "version":1.4,
 "width":20
}
```

**Comment by seancze** [comment]
Yup, having the same issue currently. Would be great if there was a fix for this!

**Comment by RafaelBarbosatec** [comment]
Oops. I will be analyzing this problem. Thanks for opening this issue

**Comment by RafaelBarbosatec** [comment]
Fix available in version 0.6.17.
